architecture/networking/routes.adoc

```diff
@@ -604,6 +604,7 @@ For all the items outlined in this section, you can set annotations on the
 |`*haproxy.router.openshift.io/cookie_name*`| Specifies an optional cookie to be used for
 this route. The name must consist of any combination of upper and lower case letters, digits, "_",
 and "-". The default is the hashed internal key name for the route. |
+|`*haproxy.router.openshift.io/pod-concurrent-connections*`| Sets the maximum number of connections that are allowed to a backing pod from a router.  Note: if there are multiple pods, each can have this many connections.  But if you have multiple routers, there is no coordination among them, each may connect this many times. If not set, there is no limit. |
```

Comment:
I just curious about what's the behaviour if set it to "0" ?  will drop any connections or allow all connections (no limit) ?
I've checked the haproxy doc: https://cbonte.github.io/haproxy-dconv/1.5/configuration.html#5.2-maxconn and it says: 
```
The default value is "0" which means unlimited.
```
so looks "not set" and "set to 0" are equivalent, we should add this to doc.

Reply:
@knobunc ^ thoughts?